### PR TITLE
Port a bug fix for GetPathRelativeTo over from Terragrunt

### PR DESCRIPTION
**Pull request by infraredgirl**
Original bug fix: https://github.com/gruntwork-io/terragrunt/pull/100

Part of #75.

### CODE REVIEW

File: files/paths.go [hunk 1]
```diff
@@ -68,6 +70,13 @@ func Grep(regex *regexp.Regexp, glob string) (bool, error) {
 
 // Return the relative path you would have to take to get from basePath to path
 func GetPathRelativeTo(path string, basePath string) (string, error) {
+	if path == "" {
```
Comment (marinalimeira) [inline files/paths.go:73]: Can you add test cases for this? [There are already some tests for this function](https://github.com/gruntwork-io/go-commons/blob/b3539bcc72ba2760b2e9f7546f40c6bfa78cb499/files/paths_test.go#L9-L33).

Comment (infraredgirl) [reply files/paths.go:73]: Sure! Done in https://github.com/gruntwork-io/go-commons/pull/76/commits/54f2f2b36cebd0e410aa5797fb7ac79c355d6bff.

**Review by marinalimeira — APPROVED** [review]
LGTM!

**Comment by infraredgirl** [comment]
Thanks - merging!

**Pull request merged**